tests/unit/modify.test.ts

```diff
@@ -40,7 +40,7 @@ describe('Application entry', () => {
     });
 
     it('GIVEN an event WHEN the environment variable NO_MODIFY is truthy THEN a callback result is returned AND the eventHandler is not called', async () => {
```

Comment:
The test description was kinda wrong to begin with, but probably needs changing even more now